docs/myriad-remote-distribution-configuration.md

```diff
@@ -12,29 +12,35 @@ your installation.
 
 First configure the Resource Manager as normal.
 
-At this point you build myriad with the commands:
-```Shell
+At this point, from the project root you build myriad with the commands:
+
+```
 ./gradlew build  
-./gradlew capsuleExecutor  
 ```
+
 and copy the jars and config onto your yarn classpath:
-```Shell
-cp build/libs/*.jar /opt/hadoop-2.5.0/share/hadoop/yarn/lib/
-cp src/main/resources/myriad-config-default.xml
+
 ```
-You will also need to place `myriad-executor-0.0.1.jar` in hdfs
-```Shell
-hadoop fs -put build/libs/myriad-executor-0.0.1.jar /dist
+cp myriad-scheduler/build/libs/*.jar /opt/hadoop-2.5.0/share/hadoop/yarn/lib/
+cp myriad-scheduler/src/main/resources/myriad-config-default.yml /opt/hadoop-2.5.0/share/hadoop/yarn/lib/
+```
+
+You will also need to place `myriad-executor-runnable-0.0.1.jar` in hdfs
+
 ```
+hadoop fs -put myriad-executor/build/libs/myriad-executor-runnable-0.0.1.jar /dist
```

Comment:
probably should be consistant with myriad-executor-runnable-0.0.1.jar and myriad-executor-runnable-xxx.jar throughout documentation.  That's not on you though.